subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -626,7 +626,7 @@
    * </pre></blockquote>
    * <img src="doc-files/recover-1.png" height="90" width="462"/>
    * <p>
-   * Note that task cancellation is not considered to be a failure.
+   * Note that task cancellation is not considered to be a failure to this method.
```

Comment:
I don't think this change is needed. The behavior of cancellation not being considered a failure is not specific to this method.

Reply:
In fact cancelled task is failed, right? It contains EarlyFinishException. I think this confuses people. Also in the user guide it compares cancellation to something like "failure" : https://github.com/linkedin/parseq/wiki/User%27s-Guide#cancelling

I personally think we should treat cancellation as special type of failure that .recover, .recoverWith(), .onFailure() don't care about (just like the doc said).

Reply:
I see @junchuanwang 's point, it is indeed confusing, especially we have several methods that treats `EarlyFinishException` differently.